src/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Twitter Clone</title>
+</head>
+
+<body>
+  <div class="grid-container">
+    <!-- Navbar -->
+    <section class="nav-bar">
+      <div class="logo"><img src="src/assets/icons/twitter.svg"></div>
+
+      <div class="navigation">
+        <div class="nav-grid-container">
```

Comment:
Would it not be better to wrap everything in this container with the nav tag rather than the individual elements inside? Or were you looking to achieve something different here?

Reply:
this just made it easier to visualise when working through it! I will probably change it now haha

Reply:
Haha good stuff, I thought there might've been some crazy technique by doing it like that that I just didn't know about 😂